services/cargohold/package.yaml

```diff
@@ -30,17 +30,23 @@ dependencies:
 - safe >=0.3
 - text >=1.1
 - transformers >=0.3
+- transformers-base >=0.3
 - yaml >=0.8
 - imports
 library:
   source-dirs: src
   dependencies:
   - base >=4 && <5
+  - amazonka >=1.3.7
+  - amazonka-core >=1.3.7
+  - amazonka-s3 >=1.3.7
+  - amazonka-cloudfront >=1.3.7
   - attoparsec >=0.12
   - auto-update >=0.1.4
-  - aws >=0.18
   - byteable >=0.1
   - base16-bytestring >=0.1
+  - blaze-builder >=0.3
+  - binary >=0.8
```

Comment:
Nit-pick: I think the only thing used from `binary` is `toLazyByteString`, which I think is just a re-export from `bytestring`.

Reply:
Good spot. We also do this in a few other places, so I cleaned them up, too.
https://github.com/wireapp/wire-server/pull/1157/commits/0bfad0234d36b3830174382a945bbc6dfeeea9ef